File: project-euler/cpp/problem_1.cpp
#include <iostream>
using namespace std;

int sum_all_multiples_of_3_or_5(int n){
    int sum_value = 0;
    for(int i=0; i<n; i++){
        if(i%3 == 0 || i%5 == 0){
            sum_value += i;
        }
    }
    return sum_value;
}

int main()
{
  cout << sum_all_multiples_of_3_or_5(1000);
  return 0;
}

File: project-euler/python/problem_1.py
def sum_all_multiples_of(n: int, a, b) -> int:
    sum_value = 0
    for i in range(1, n):
        if i%a == 0 or i%b == 0:
            sum_value += i
    return sum_value

if __name__ == '__main__':
    print('sum of all multiples of 3 or 5 from 1 to 999')
    print(sum_all_multiples_of(n=1000, a=3, b=5))